### ISSUE TITLE
AM ne fonctionnent plus

### ISSUE THREAD

**Issue by DrFloW71** [Refhi/Weda-Helper#246]
Probablement suite à la MaJ de WEDA connect les AM bloquent sur le pdf et ne s'impriment plus tous seuls.

**Comment by Refhi** [comment]
Oui, j'ai vu ça, merci pour le ticket, je me penche dessus à l'occasion.
Je pense que c'est juste l'url qui pose problème depuis la mise à jour de chrome (c'est plus http... mais blob:...) faut qu'on revoit ça, mais c'est une galère sans nom à tester parceque je ne peut le faire QUE en consultation pendant la réalisation d'un arrêt de travail >.<

**Comment by Refhi** [comment]
Après avoir fouillé un peu, l'extension ne se charge plus sur la fenêtre de prévisualisation des AT...

J'en discute demain avec Weda voire s'ils ont une idée, mais c'est probablement bloqué pour l'instant...

**Comment by Refhi** [comment]
En attente d'un retour de Weda

**Comment by Refhi** [comment]
Je tente un fix en attendant d'éventuelles modifs de Weda.

**Comment by Refhi** [comment]
Réparé dans la 2.8 !
